ch_collections/heritage_services/roles/nfs/tasks/main.yml

```diff
@@ -4,6 +4,7 @@
   include_vars: "{{ item }}"
   with_first_found:
     - files:
+        - "{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
```

Comment:
Add `ansible_distribution` as a priority over `ansible_os_family`, allows us to differentiate between different RHEL based OS's (e.g. Amazon linux 2 and Centos 7)